protocol/x/clob/keeper/random.go

```diff
@@ -0,0 +1,16 @@
+package keeper
+
+import (
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// GetPseudoRand returns a random number generator seeded with a pseudorandom seed.
+// The seed is based on the previous block timestamp.
+func (k *Keeper) GetPseudoRand(ctx sdk.Context) *rand.Rand {
```

Comment:
Suggestion, up to you: should this function be unit tested? Maybe a simple unit test for determinism (just verify that same block info produces same values, and updating it produces diff values)?

Reply:
math/rand should have pretty comprehensive unit tests for determinism. personally don't feel like it's necessary.